lib/rpmsg/rpmsg_virtio.c

```diff
@@ -251,7 +251,7 @@ static int _rpmsg_virtio_get_buffer_size(struct rpmsg_virtio_device *rvdev)
 		 * so get the buffer size from the virtqueue.
 		 */
 		length =
-		    (int)virtqueue_get_desc_size(rvdev->svq) -
+		    virtqueue_get_desc_size(rvdev->svq) -
```

Comment:
this returns unsigned, it can be compilation error if assign unsigned to int.

Reply:
Fix